src/docs/patterns/lists.mdx

```diff
@@ -60,7 +197,142 @@ Occasionally, you may want users to be able to re-order the items in a list manu
+
+### Actions on Selected Items
+
+<ImageGrid images={[MultiSelectImage]} regularWidth />
+<DemoCard repository={'multiselect-list'} angular={'https://stackblitz.com/github/pxblue/angular-design-patterns?file=src%2Fapp%2Fpages%2Flist%2Fmultiselect-list%2Fmultiselect-list.component.ts'} react={'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/lists/multiselect-list/index.tsx&initialpath=multiselect-list'} ionic reactNative />
+
+You may want to take action on several list items at once, but not all. The easiest way to do this is through a multi-select list. You may optionally add a highlight color for the selected items.
+
+## Responsive Design
+
+When it comes to responsive design and progressive disclosure, it can be tricky to fit all the information from a list onto a smaller screen. Here are some best practices on designing for mobile phone:
+
+-   If a list item has too many elements in it and doesn't fit, try converting the item into a summary card, with a "view more" option to view the full data.
+-   Avoid using tables.
+-   Try adding gestures (long-press, swipe-left) that fits with the phone's OS ecosystem.
+
+<Divider />
+
+# Design Specifications
+
+<ImageGrid images={[Spec1Image, Spec2Image, Spec3Image, Spec4Image, Spec5Image, SpecTypographyImage]} regularWidth />
+<MaterialDesignDescription title={'Lists'} description={`For additional instructions, refer to Material Design's design specs`} url={'https://material.io/components/lists#specs'} />
+<MaterialDesignDescription title={'Data Tables'} description={`For additional instructions, refer to Material Design's design specs`} url={'https://material.io/components/data-tables#specs'} />
+<Divider />
+
+# Developers
+
+Use the following components to implement this pattern:
+
+**Angular** and **Ionic**:
+
+-   Angular Material
+    -   [List](https://material.angular.io/components/list/overview)
+    -   [Table](https://material.angular.io/components/table/overview)
+-   @pxblue/angular-components
+    -   [Info List Item](https://pxblue-components.github.io/angular/?path=/info/components-info-list-item--readme)
+    -   [List Item Tag](https://pxblue-components.github.io/angular/?path=/info/components-list-item-tag--readme)
+
+In Angular, the CDK (Component Developer's Kit) greater than 7.0 is required for the drag-and-drop feature.
+
+**React**:
+
+-   Material UI
+    -   [List](https://material-ui.com/api/list/)
+    -   [List Item](https://material-ui.com/api/list-item/)
+-   @pxblue/angular-components
+    -   [Info List Item](https://pxblue-components.github.io/react/?path=/info/components-info-list-item--get-read-me-story)
+    -   [List Item Tag](https://pxblue-components.github.io/react/?path=/info/components-list-item-tag--get-read-me-story)
+
+**React Native**:
+
+-   React-Native
+    -   [Flat List](https://reactnative.dev/docs/using-a-listview)
+-   React Native Elements
```

Comment:
Take out React Native elements...it's not our recommended component library anymore (just RNP now)

Reply:
Ok. I got the idea from our React Native component library showcase. Should I log an issue in that repo?

Reply:
I believe we have a story in JIRA to overhaul that showcase and make it a shared submodule (R18). If you want to check the criteria for that and make sure it says to remove RNE, that should suffice. RNE was previously our preferred component library, but RNP has pulled ahead and is more material-y.